drivers/serial/Kconfig.stm32

```diff
@@ -7,10 +7,12 @@
 
 menuconfig UART_STM32
 	bool "STM32 MCU serial driver"
-	default n
+	depends on SOC_FAMILY_STM32
 	select SERIAL_HAS_DRIVER
 	select SERIAL_SUPPORT_INTERRUPT
-	depends on SOC_FAMILY_STM32
+	select USE_STM32_LL_USART
```

Comment:
Selecting USE_STM32_LL_USART does not look related to LPUART. Is that a separate fix?

Reply:
Yes, it's just a fixing things around. I can separate it too.

Reply:
Though it should be fixed before adding LPUART.

Reply:
I don't think this was actually required for U(S)ART driver.
Stm32 serial driver only used LL headers until now, which do not require this flag.